recipes/ocaml/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export LC_ALL=C
+./configure -prefix $PREFIX
+make world.opt
+umask 022
+make install
```

Comment:
is there something like `make test` which we can run?

Reply:
I'll check - if there is but its long, is that still OK for conda-forge?

(I know BioConda avoids including long tests)

Reply:
This really depends what long means. We are also here bound to what the CI is restricting us to.